### PR TITLE
Document new API for Node analyzers

### PR DESCRIPTION
**Pull request by lukas-vlcek**
### Description

This commit introduces documentation for new extension of Nodes API that exposes names of analysis components available on cluster node(s).

This commit also contains additional changes:

- It makes strict distinction between terms: "token" and "term".
- It replaces the term "Normalize" in analysis part because it has special meaning in this context
- It introduces a dedicated pages for Normalization (which is a specific type of analysis)

In addition to this, this PR contains two commits where the `1c13038` commit fixes some broken links on the plugin page and might be a good candidate for pack-porting.

### Issues Resolved

This issue is part of https://github.com/opensearch-project/OpenSearch/pull/10296
Do not merge it if ^^ is not merged as well.

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by lukas-vlcek** [comment]
Note: Depending on when (or if) the related PR in OpenSearch is merged then we need to update the `v2.11.??` in the documentation.

**Comment by hdhalter** [comment]
Thanks, @lukas-vlcek ! This is addressing this issue, right? https://github.com/opensearch-project/documentation-website/issues/5426. Is it ready for a doc review? 

We can look at the additional fixes, and backport them. Thanks for catching and fixing them!

**Comment by lukas-vlcek** [comment]
> Thanks, @lukas-vlcek ! This is addressing this issue, right? #5426. Is it ready for a doc review?

@hdhalter Please see my response https://github.com/opensearch-project/documentation-website/issues/5426#issuecomment-1919374135 It is currently not part of `_cat` API.

File: _analyzers/index.md [hunk 1]
```diff
@@ -15,16 +15,24 @@ redirect_from:
 
 # Text analysis
 
-When you are searching documents using a full-text search, you want to receive all relevant results and not only exact matches. If you're looking for "walk", you're interested in results that contain any form of the word, like "Walk", "walked", or "walking." To facilitate full-text search, OpenSearch uses text analysis.
+When you are searching documents using a full-text search, you want to receive all relevant results. If you're looking for "walk", you're interested in results that contain any form of the word, like "Walk", "walked", or "walking". To facilitate full-text search, OpenSearch uses text analysis.
 
-Text analysis consists of the following steps:
+The objective of the text analysis is to break down the unstructured free-text content of the source document into a sequence of terms, which are then stored in an inverted index. Subsequently when a similar text analysis is applied to a user's query, the resulting sequence of terms facilitates the matching of relevant source documents.
```
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/index.md:20]: @kolchfa-aws If I remember correctly I was getting rejection from the DocBot for leaving comma after "Subsequently,". But let me try again...

[the remark this answers is not in the document]
Comment (kolchfa-aws) [reply _analyzers/index.md:20]: @lukas-vlcek I confirmed with our editor that the comma is necessary in this case. The bot is not too intelligent :) In any case, we'll have an editorial review on this PR so these things will be fixed by our editor @natebower. Thanks!

[the remark this answers is not in the document]
Comment (kolchfa-aws) [reply _analyzers/index.md:20]: Update: we've removed the rule from the bot (#6323)

**Comment by hdhalter** [comment]
> > Thanks, @lukas-vlcek ! This is addressing this issue, right? #5426. Is it ready for a doc review?
> 
> @hdhalter Please see my response [#5426 (comment)](https://github.com/opensearch-project/documentation-website/issues/5426#issuecomment-1919374135) It is currently not part of `_cat` API.

Thanks, @lukas-vlcek, I closed the issue related to _cat API. For this node API PR, is it going out in 2.12?

File: _analyzers/index.md [hunk 2]
```diff
@@ -15,16 +15,24 @@ redirect_from:
...
 
-1. _Tokenize_ text into terms: For example, after tokenization, the phrase `Actions speak louder than words` is split into tokens `Actions`, `speak`, `louder`, `than`, and `words`.
-1. _Normalize_ the terms by converting them into a standard format, for example, converting them to lowercase or performing stemming (reducing the word to its root): For example, after normalization, `Actions` becomes `action`, `louder` becomes `loud`, and `words` becomes `word`.
+From a technical point of view, the text analysis process consists of several steps, some of which are optional.
+
+1. Before the free-text content can be broken down into individual words it may be beneficial to treat it at the character level. The primary aim of this optional step would be to help the tokenizer (the subsequent stage in the analysis process) generating better tokens. This can include removal of markup tags (such as HTML) or handling a specific character patterns (like replacing emoji &#x1F642; with a text `:slightly_smiling_face:`).
```
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/index.md:24]: > ... it may be beneficial to "apply" it at the character level ...

I think we need to find a better formulation. What I try to say here is that it might be necessary or useful to "heal" the text at the character level. It is similar to "normalization" that I talk about later. We are not dealing with individual words yet but we may need to treat characters or patterns or characters.

My original formulation ("treat") was not good at all. But using only "apply" does not IMO do the right job as well. We need to say WHAT to apply. Or use a different word what has the meaning of "healing" or something like that ... sorry, I do not have better explanation, I know it is far from ideal.

Do you have any ideas please?

File: _analyzers/index.md [hunk 3]
```diff
@@ -54,6 +64,317 @@ Analyzer | Analysis performed | Analyzer output
 
 If needed, you can combine tokenizers, token filters, and character filters to create a custom analyzer.
 
+With the introduction of OpenSearch `v2.11.??`, you can retrieve a comprehensive list of all available text analysis components using [Nodes Info]({{site.url}}{{site.baseurl}}/api-reference/nodes-apis/nodes-info/). This can be helpful when building custom analyzers, especially in cases where you need to recall the component's name or identify associated analysis plugin this component is part of.
```
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/index.md:67]: FYI, I am going to apply `v2.12.0` for now.

File: _analyzers/normalizers.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: Normalizers
+nav_order: 100
+---
+
+# Normalizers
+
+A Normalizer is similar to Analyzer except it does not contain tokenizer, and it can only include specific type of Character filters and Token filters. These filters can do only character level operations, such as character or pattern replacement, but can not operate on the whole token level, this means that replacing the token with synonym or stemming is not supported.
```
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/normalizers.md:9]: Here I am going to propose changes in the beginning of the sentence. Please check when the PR is updated.

File: _analyzers/normalizers.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+### Plugin `analysis-kuromoji`
+
+Character filters: `normalize_kanji`, `normalize_kana`
+
+### Plugin `analysis-nori`
+
+Character filters: `normalize_kanji`, `normalize_kana`
+
+This list of filters covers only analysis components found in [additional plugins]({{{site.url}}{{site.baseurl}}/install-and-configure/plugins/#additional-plugins) that are part of the core OpenSearch repository.
```
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/normalizers.md:98]: Good catch!

File: _analyzers/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kolchfa-aws) [reply _analyzers/index.md]: How about "process" or "refine" the text at a character level? Does this work?

[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/index.md]: Yep, I take:
> ... it may be beneficial to refine the text at a character level.

File: _analyzers/normalizers.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/normalizers.md]: FYI: PR is updated.

**Comment by lukas-vlcek** [comment]
@kolchfa-aws Thanks for detailed review. I updated the PR and marked all comments as resolved.

**Comment by kolchfa-aws** [comment]
Thank you, @lukas-vlcek! I'll move the PR to editorial review.

File: _analyzers/normalizers.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+  "query": {
+    "term": {
+      "approach": "Naïve"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+To understand why, try to see the effect of the normalizer:
```
Comment (natebower) [inline _analyzers/normalizers.md:77]:
```suggestion
To understand why, consider the effect of the normalizer:
```

File: _analyzers/normalizers.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+```json
+GET /sample-index/_analyze
+{
+  "normalizer" : "normalized_keyword",
+  "text" : "Naïve"
+}
+```
+
+Internally, a normalizer accepts only filters that are instances of either `NormalizingTokenFilterFactory` or `NormalizingCharFilterFactory`. The following is a list of compatible filters found in modules and plugins that are part of the core OpenSearch repository:
```
Comment (natebower) [inline _analyzers/normalizers.md:86]:
```suggestion
Internally, a normalizer accepts only filters that are instances of either `NormalizingTokenFilterFactory` or `NormalizingCharFilterFactory`. The following is a list of compatible filters found in modules and plugins that are part of the core OpenSearch repository.
```

File: _analyzers/normalizers.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+### Module `common-analysis`
```
Comment (natebower) [inline _analyzers/normalizers.md:88]: @kolchfa-aws As with the "Step" headings in an earlier PR, the word order of this and the following three headings is confusing. Please revise.

File: _analyzers/normalizers.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+
+This module does not require installation, it is available by default.
```
Comment (natebower) [inline _analyzers/normalizers.md:90]:
```suggestion
This module does not require installation; it is available by default.
```

File: _analyzers/normalizers.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+
+Character filters: `pattern_replace`, `mapping`
+
+Token filters: `arabic_normalization`, `asciifolding`, `bengali_normalization`, `cjk_width`, `decimal_digit`, `elision`, `german_normalization`, `hindi_normalization`, `indic_normalization`, `lowercase`, `persian_normalization`, `scandinavian_folding`, `scandinavian_normalization`, `serbian_normalization`, `sorani_normalization`, `trim`, `uppercase`
+
+### Plugin `analysis-icu`
+
+Character filters: `icu_normalizer`
+
+Token filter: `icu_normalizer`, `icu_folding`, `icu_transform`
```
Comment (natebower) [inline _analyzers/normalizers.md:100]:
```suggestion
Token filters: `icu_normalizer`, `icu_folding`, `icu_transform`
```

File: _analyzers/normalizers.md [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+
+### Plugin `analysis-kuromoji`
+
+Character filters: `normalize_kanji`, `normalize_kana`
+
+### Plugin `analysis-nori`
+
+Character filters: `normalize_kanji`, `normalize_kana`
+
+This list of filters covers only analysis components found in [additional plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/#additional-plugins) that are part of the core OpenSearch repository.
```
Comment (natebower) [inline _analyzers/normalizers.md:110]:
```suggestion
These lists of filters include only analysis components found in the [additional plugins]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/#additional-plugins) that are part of the core OpenSearch repository.
```

File: _install-and-configure/plugins.md [hunk 12]
```diff
@@ -85,13 +85,13 @@ opensearch-node1 opensearch-notifications-core        2.0.1.0
 
 There are three ways to install plugins using the `opensearch-plugin`:
 
-- [Install a plugin by name]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#install-a-plugin-by-name)
-- [Install a plugin by from a zip file]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#install-a-plugin-from-a-zip-file)
-- [Install a plugin using Maven coordinates]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#install-a-plugin-using-maven-coordinates)
+- [Install a plugin by name](#install-a-plugin-by-name)
+- [Install a plugin by from a zip file](#install-a-plugin-from-a-zip-file)
```
Comment (natebower) [inline _install-and-configure/plugins.md:89]:
```suggestion
- [Install a plugin from a ZIP file](#install-a-plugin-from-a-zip-file).
```

File: _install-and-configure/plugins.md [hunk 13]
```diff
@@ -85,13 +85,13 @@ opensearch-node1 opensearch-notifications-core        2.0.1.0
 
 There are three ways to install plugins using the `opensearch-plugin`:
 
-- [Install a plugin by name]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#install-a-plugin-by-name)
-- [Install a plugin by from a zip file]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#install-a-plugin-from-a-zip-file)
-- [Install a plugin using Maven coordinates]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#install-a-plugin-using-maven-coordinates)
+- [Install a plugin by name](#install-a-plugin-by-name)
```
Comment (natebower) [inline _install-and-configure/plugins.md:88]:
```suggestion
- [Install a plugin by name](#install-a-plugin-by-name).
```

File: _install-and-configure/plugins.md [hunk 14]
```diff
@@ -85,13 +85,13 @@ opensearch-node1 opensearch-notifications-core        2.0.1.0
...
+- [Install a plugin by from a zip file](#install-a-plugin-from-a-zip-file)
+- [Install a plugin using Maven coordinates](#install-a-plugin-using-maven-coordinates)
```
Comment (natebower) [inline _install-and-configure/plugins.md:90]:
```suggestion
- [Install a plugin using Maven coordinates](#install-a-plugin-using-maven-coordinates).
```

File: _install-and-configure/plugins.md [hunk 15]
```diff
@@ -85,13 +85,13 @@ opensearch-node1 opensearch-notifications-core        2.0.1.0
...
 
 ### Install a plugin by name:
```
Comment (natebower) [inline _install-and-configure/plugins.md:92]:
```suggestion
### Install a plugin by name
```

File: _install-and-configure/plugins.md [hunk 16]
```diff
@@ -85,13 +85,13 @@ opensearch-node1 opensearch-notifications-core        2.0.1.0
...
 
-For a list of plugins that can be installed by name, see [Additional plugins]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#additional-plugins).
+For a list of plugins that can be installed by name, see [Additional plugins](#additional-plugins).
 
 #### Usage:
```
Comment (natebower) [inline _install-and-configure/plugins.md:96]:
```suggestion
#### Usage
```

Comment (natebower) [inline _install-and-configure/plugins.md:96]: Global: Please remove terminal punctuation from headings.

**Review by natebower — COMMENTED** [review]
@kolchfa-aws @lukas-vlcek Please see my comments and changes and let me know if you have any questions. Thanks!

File: _analyzers/index.md [hunk 17]
```diff
@@ -15,16 +15,24 @@ redirect_from:
...
-1. _Normalize_ the terms by converting them into a standard format, for example, converting them to lowercase or performing stemming (reducing the word to its root): For example, after normalization, `Actions` becomes `action`, `louder` becomes `loud`, and `words` becomes `word`.
+From a technical point of view, the text analysis process consists of several steps, some of which are optional:
+
+1. Before the free-text content can be split into individual words, it may be beneficial to refine the text at a character level. The primary aim of this optional step is to help the tokenizer (the subsequent stage in the analysis process) generate better tokens. This can include removal of markup tags (such as HTML) or handling a specific character patterns (like replacing the &#x1F642; emoji with the text `:slightly_smiling_face:`).
+
+2. The next step is to split the free-text into individual words---_tokens_.. This is a job of a _tokenizer_. For example, after tokenization, the sentence `Actions speak louder than words` is split into tokens `Actions`, `speak`, `louder`, `than`, and `words`.
```
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/index.md:26]: "Free-text" or "free text", I am fine with both but we need to be consistent. Please see line 24. it uses "free-text". So we need to decide which version we want to go with.

[the remark this answers is not in the document]
Comment (natebower) [reply _analyzers/index.md:26]: Unless "free-text" is a compound word that is always hyphenated, it should be hyphenated when used a compound modifier preceding the noun it modifies (as on line 24) but not hyphenated otherwise: "This is free-text content" but "The content consisted of free text."

File: _analyzers/index.md [hunk 18]
```diff
@@ -15,16 +15,24 @@ redirect_from:
...
+
+3. The last step is to process individual tokens by applying a series of token filters. The aim is to convert each token into a predictable form that is directly stored in the index, for example, by converting them to lowercase or performing stemming (reducing the word to its root). For example, the token `Actions` becomes `action`, `louder` becomes `loud`, and `words` becomes `word`.
+
+Although the terms ***token*** and ***term*** may sound similar and are occasionally used interchangeably, it is helpful to understand the difference between the two. In the context of Apache Lucene®, each holds a distinct role. A ***token*** is created by a tokenizer during text analysis and often undergoes a number of additional modifications as it passes through the chain of token filters. Each token is associated with metadata that can be further used during the text analysis process. A ***term*** is a data value that is directly stored in the inverted index and is associated with a lot less metadata. During search, matching operates at the term level.
```
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/index.md:30]: @natebower I am checking Apache guide on project name usage https://www.apache.org/foundation/marks/guide#naming and it seems that dropping "(R)" should be fine if:

> The first and most prominent mentions must use the full form: "Apache Hadoop®" of the name for any individual usage (webpage, handout, slides, etc.) Depending on the context and writing style, you should use the full form of the name sufficiently often to ensure that readers clearly understand the association of both the Hadoop project and the Hadoop software product to the ASF as the parent organization.
>
> Later in each specific document you may use the bare form of the name, i.e. just Hadoop, as best suits your writing style.

It seems legit to drop the (R) here. 👍

[the remark this answers is not in the document]
Comment (natebower) [reply _analyzers/index.md:30]: @lukas-vlcek The ® is mostly for Apache to provide notice to the public that a trademark or service mark is registered. Although the symbols ® and ™ (for registered and unregistered trademarks, respectively) often accompany trademark names on product packaging and in promotional material, there is no legal requirement to use these symbols, and they should be omitted wherever possible.

[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/index.md:30]: @natebower Sure, but still you do not want to give anybody an opportunity to publicly point at you or your employer for not using legal attributions correctly (unfortunately this can happen).

File: _install-and-configure/plugins.md (hunk 16 above)
Comment (lukas-vlcek) [reply _install-and-configure/plugins.md:96]: I am going to take care of it in `_install-and-configure/plugins.md` document.
If heading creates an URL link with `#page_anchors` then punctuation does not feel right (aside from other conventions).

Comment (kolchfa-aws) [reply _install-and-configure/plugins.md:96]: Thank you for doing this, @lukas-vlcek! I know this was not part of your changes and we appreciate you going an extra mile!

File: _analyzers/normalizers.md [hunk 19]
```diff
@@ -0,0 +1,111 @@
...
+GET /sample-index/_analyze
+{
+  "normalizer" : "normalized_keyword",
+  "text" : "Naïve"
+}
+```
+
+Internally, a normalizer accepts only filters that are instances of either `NormalizingTokenFilterFactory` or `NormalizingCharFilterFactory`. The following is a list of compatible filters found in modules and plugins that are part of the core OpenSearch repository.
+
+### Module `common-analysis`
```
Comment (kolchfa-aws) [inline _analyzers/normalizers.md:88]:
```suggestion
### The `common-analysis` module
```

File: _analyzers/normalizers.md [hunk 20]
```diff
@@ -0,0 +1,111 @@
...
+
+This module does not require installation; it is available by default.
+
+Character filters: `pattern_replace`, `mapping`
+
+Token filters: `arabic_normalization`, `asciifolding`, `bengali_normalization`, `cjk_width`, `decimal_digit`, `elision`, `german_normalization`, `hindi_normalization`, `indic_normalization`, `lowercase`, `persian_normalization`, `scandinavian_folding`, `scandinavian_normalization`, `serbian_normalization`, `sorani_normalization`, `trim`, `uppercase`
+
+### Plugin `analysis-icu`
```
Comment (kolchfa-aws) [inline _analyzers/normalizers.md:96]:
```suggestion
### The `analysis-icu` plugin
```

File: _analyzers/normalizers.md [hunk 21]
```diff
@@ -0,0 +1,111 @@
...
+
+Character filters: `icu_normalizer`
+
+Token filters: `icu_normalizer`, `icu_folding`, `icu_transform`
+
+### Plugin `analysis-kuromoji`
```
Comment (kolchfa-aws) [inline _analyzers/normalizers.md:102]:
```suggestion
### The `analysis-kuromoji` plugin
```

File: _analyzers/normalizers.md [hunk 22]
```diff
@@ -0,0 +1,111 @@
...
+
+Character filters: `normalize_kanji`, `normalize_kana`
+
+### Plugin `analysis-nori`
```
Comment (kolchfa-aws) [inline _analyzers/normalizers.md:106]:
```suggestion
### The `analysis-nori` plugin
```

File: _install-and-configure/plugins.md (hunk 16 above)
Comment (lukas-vlcek) [reply _install-and-configure/plugins.md]: @kolchfa-aws I added these changes to my second commit (that was fixing some broken links in plugins page) so that they can be cherry-picked easily.

File: _analyzers/index.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (lukas-vlcek) [reply _analyzers/index.md]: @natebower You are right, let's go with "free text". Let me push update...

**Comment by kolchfa-aws** [comment]
@lukas-vlcek Just a couple of more suggestions to reword the headings. Thanks!

File: _analyzers/normalizers.md (hunk 8 above)
Comment (lukas-vlcek) [reply _analyzers/normalizers.md]: @kolchfa-aws I think we can resolve this one too now.

**Comment by lukas-vlcek** [comment]
@kolchfa-aws @natebower Thanks for all the reviews and feedback!
I think the PR is ready and can be merged once https://github.com/opensearch-project/OpenSearch/pull/10296 is merged.

**Comment by kolchfa-aws** [comment]
Thank you, @lukas-vlcek! I will mark is as ready to merge and we'll merge this PR when the dev PR is merged.

**Comment by hdhalter** [comment]
@lukas-vlcek - Should we put this back in tech review (or draft) until you have a chance to modify it?

**Comment by lukas-vlcek** [comment]
@hdhalter 
I am going to update this PR. It will have two parts. One part will be general and can go upstream as soon as the docs review is finished. The second part will be relevant only to a new functionality and that can go in after the relevant code PR is merged. I will leave it up to you how you want to process this from the review process point – if you think putting it back to draft mode is way to go then let's do it. If you think it is better to handle it differently please let me know.

To me it sounds like I should open a new PR for the general part (this can go in anytime). And then I will update this PR and leave only content that is relevant to a new functionality.

**Comment by hdhalter** [comment]
> @hdhalter I am going to update this PR. It will have two parts. One part will be general and can go upstream as soon as the docs review is finished. The second part will be relevant only to a new functionality and that can go in after the relevant code PR is merged. I will leave it up to you how you want to process this from the review process point – if you think putting it back to draft mode is way to go then let's do it. If you think it is better to handle it differently please let me know.
> 
> To me it sounds like I should open a new PR for the general part (this can go in anytime). And then I will update this PR and leave only content that is relevant to a new functionality.

Your suggestion to put this in draft and create a new PR for general changes sounds good to me. Thanks so much, really appreciate your support!

**Comment by lukas-vlcek** [comment]
@hdhalter Please see https://github.com/opensearch-project/documentation-website/pull/6415
That is a new PR that contains all changes except those related to a new functionality (i.e. node analyzers).

I am going to update this PR and it will contain only the node analyzers part and can be merged once the upstream code PR is merged as well (that code PR needs similar update, I will move changes note directly relevant to node changes to separated code PRs).

**Comment by lukas-vlcek** [comment]
I updated this PR. Now, it contains only part relevant to the new functionality. All the other changes has been already merged in #6415.

I have only two comments:

1) The text states that the functionality was (will be) introduced in `2.12.1`. This needs to be verified before merging.
2) I am considering making the JSON response example block closed by default. It is quite a long output. I think I can only remove the `open` attribute of the `detail` element. Let me know what you think.

//cc @kolchfa-aws @hdhalter

**Comment by kolchfa-aws** [comment]
@lukas-vlcek Thanks for doing this! We'll update the version text before merging. I agree with making long requests/responses closed by default. You can change "open" to "closed' to achieve that.

**Comment by lukas-vlcek** [comment]
@kolchfa-aws 
> You can change "open" to "closed' to achieve that.

I was inspecting the life HTML document in chrome and it seemed that there is no `closed` option added to the `details` element when it is closed. There was either `open` or nothing... just nitpicking... the [FORMATTING_GUIDE.md#collapsible-blocks](https://github.com/opensearch-project/documentation-website/blob/main/FORMATTING_GUIDE.md#collapsible-blocks) does not specify this situation either.

**Comment by lukas-vlcek** [comment]
BTW, WRT the version, the feature seems to be scheduled for 2.13, see https://github.com/opensearch-project/OpenSearch/issues/5481#event-11779857606

**Comment by kolchfa-aws** [comment]
You're right. It's closed by default and you add the `open` state to have it open. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details. I'll add it to the formatting guide.

**Comment by lukas-vlcek** [comment]
@kolchfa-aws I updated the `details` element, removed the `open` attribute.

**Comment by kolchfa-aws** [comment]
@lukas-vlcek Do you think this feature will be included in 2.13?

**Comment by lukas-vlcek** [comment]
@kolchfa-aws Unfortunately not, I did not make it. Sorry about that. Let's plan it for 2.14.

**Comment by Naarcha-AWS** [comment]
@lukas-vlcek: Is this still targeting 2.14?

**Comment by kolchfa-aws** [comment]
@Naarcha-AWS I'm back so I'll take this PR from here. Thank you!

**Comment by kolchfa-aws** [comment]
@lukas-vlcek Do you know if this is going into 2.15?

**Comment by hdhalter** [comment]
Assuming this is not going in 2.15, since @reta removed the 2.15 label from https://github.com/opensearch-project/OpenSearch/pull/10296.
